src/librados/librados_c.cc

```diff
@@ -91,7 +99,7 @@ static CephContext *rados_create_cct(const char * const clustername,
   return cct;
 }
 
-extern "C" int rados_create(rados_t *pcluster, const char * const id)
+extern "C" int __rados_create(rados_t *pcluster, const char * const id)
```

Comment:
in C standard, 

> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

see https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html

Reply:
Fixed